core/src/main/java/org/hibernate/ogm/dialect/spi/TransactionContext.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.dialect.spi;
+
+/**
+ * @author Davide D'Alto
+ */
+public interface TransactionContext {
+
+	Long getTransactionId();
```

Comment:
It's guess-work at this point, but I feel limiting it to `Long` might be too tight. I'd make it `String` at least, better yet `Object` to be prepared for other implmentations.